src/components/graph/graph.builder.js

```diff
@@ -206,7 +206,7 @@ function buildNodeProps(node, config, nodeCallbacks = {}, highlightedNode, highl
         label,
         opacity,
         overrideGlobalViewGenerator: !node.viewGenerator && node.svg,
-        renderLabel: config.node.renderLabel,
+        renderLabel: node.renderLabel || config.node.renderLabel,
```

Comment:
This will work just fine, basically, if no `renderLabel` is defined at node level we fallback to global :+1: